From b93f5f4f64e7bbe2d137b41c8459a75b33d55b4a Mon Sep 17 00:00:00 2001
From: Han Gao <rabenda.cn@gmail.com>
Date: Fri, 24 Jan 2025 17:16:08 +0800
Subject: [PATCH 364/481] configs: eswin: add
 DEVFREQ_GOV_{PERFORMANCE|POWERSAVE|USERSPACE|PASSIVE}

+CONFIG_DEVFREQ_GOV_PERFORMANCE=y
+CONFIG_DEVFREQ_GOV_POWERSAVE=y
+CONFIG_DEVFREQ_GOV_USERSPACE=y
+CONFIG_DEVFREQ_GOV_PASSIVE=y

Signed-off-by: Han Gao <rabenda.cn@gmail.com>
---
 arch/riscv/configs/win2030_defconfig | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/riscv/configs/win2030_defconfig b/arch/riscv/configs/win2030_defconfig
index 7601a3cf4932..6af7ca38d0c9 100644
--- a/arch/riscv/configs/win2030_defconfig
+++ b/arch/riscv/configs/win2030_defconfig
@@ -787,6 +787,10 @@ CONFIG_ESWIN_DSP=m
 CONFIG_ESWIN_NPU=m
 CONFIG_PM_DEVFREQ=y
 CONFIG_DEVFREQ_GOV_SIMPLE_ONDEMAND=y
+CONFIG_DEVFREQ_GOV_PERFORMANCE=y
+CONFIG_DEVFREQ_GOV_POWERSAVE=y
+CONFIG_DEVFREQ_GOV_USERSPACE=y
+CONFIG_DEVFREQ_GOV_PASSIVE=y
 CONFIG_EXTCON=y
 CONFIG_MEMORY=y
 CONFIG_ESWIN_BUDDY=y
-- 
2.49.0

